packages/dds/merge-tree/src/test/revertibles.spec.ts

```diff
@@ -207,19 +203,24 @@ describe("MergeTree.Revertibles", () => {
         const ops: ISequencedDocumentMessage[] = [];
 
         const clientB_Revertibles: MergeTreeDeltaRevertible[] = [];
-        // the test logger uses these callbacks, so preserve it
-        const old = clients.B.mergeTreeDeltaCallback;
+
         const clientBDriver = createRevertDriver(clients.B);
         clientBDriver.submitOpCallback = (op) => ops.push(clients.B.makeOpMessage(op, ++seq));
-        clients.B.mergeTreeDeltaCallback = (op, delta) => {
-            old?.(op, delta);
+
+        clients.B.on("delta", (op, delta) => {
             if (op.sequencedMessage === undefined) {
-                 appendToMergeTreeDeltaRevertibles(clientBDriver, delta, clientB_Revertibles);
+                appendToMergeTreeDeltaRevertibles(clientBDriver, delta, clientB_Revertibles);
             }
-        };
+        });
         ops.push(clients.B.makeOpMessage(clients.B.annotateRangeLocal(0, 4, { test: "B" }, undefined), ++seq));
         ops.push(clients.B.makeOpMessage(clients.B.removeRangeLocal(1, 2), ++seq));
-        clients.B.mergeTreeDeltaCallback = old;
+
+        // revert to the original callback
+        clients.B.off("delta", (op, delta) => {
```

Comment:
you can't pass a new function to off, it needs to be the same function passed to on